### ISSUE TITLE
Fixing gradient backprop in #233

### ISSUE THREAD

**Issue by JeanKossaifi** [neuraloperator/neuraloperator#236]
#233 adds a pure PyTorch fallback for scatter/cluster but there is an issue with unused gradients. Opening this issue to keep track, #233 will be merged to unblock other PRs and fix CI in the meantime. @dhpitt
